index.js

```diff
@@ -1,27 +1,26 @@
 'use strict';
+const fs = require('fs');
 const uniqueRandomArray = require('unique-random-array');
 const pokemon = require('./pokemon');
-const pokemonGerman = require('./pokemon_de');
 
 exports.all = pokemon;
 exports.random = uniqueRandomArray(pokemon);
 
-function getLocalizedPokemonList(lang) {
-	let list;
-
-	switch (lang) {
-		case 'de':
-			list = pokemonGerman;
-			break;
-		default:
-			list = pokemon;
+function getLocalizedList(lang) {
+	if (!lang || lang === 'en') {
+		return pokemon;
 	}
 
-	return list;
+	try {
+		const list = fs.readFileSync(`./pokemon_${lang}.json`, 'utf8');
```

Comment:
Ok, I didn't knew that `require()` caches libraries on consecutive calls. Thanks for pointing that out. 

I will check if I can just do:
```js
const list = require(`./pokemon_${lang}`);
```